decentraland/unity-renderer
# chore: remove emotes customization feature flag

## What does this PR change?

Remotes the emote-customization feature flag, and sets it always to true


## How to test the changes?

Go to: https://play.decentraland.zone/?renderer-branch=chore/remove-emotes-customization-flag
Open the emotes wheel
Check that the Customize button is present and clickable
Assign emotes to slots and save
Reopen and verify that the emotes are assigned to the same slot

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
